pkg/sql/opt/exec/execbuilder/testdata/inverted_index

```diff
@@ -3,12 +3,49 @@
 statement ok
 CREATE TABLE d (
   a INT PRIMARY KEY,
-  b JSONB
+  b JSONB,
+  FAMILY (a,b)
 )
 
 statement ok
 CREATE INVERTED INDEX foo_inv ON d(b)
 
+query T kvtrace
+INSERT INTO d VALUES(0, '{"a": "b"}')
+----
+CPut /Table/53/1/0/0 -> /TUPLE/
+InitPut /Table/53/2/"a"/"b"/0/0 -> /BYTES/
+
+# Make sure duplicate values don't get inserted.
+query T kvtrace
+INSERT INTO d VALUES(1, '[7,0,7]')
+----
+CPut /Table/53/1/1/0 -> /TUPLE/
+InitPut /Table/53/2/Arr/0/1/0 -> /BYTES/
+InitPut /Table/53/2/Arr/7/1/0 -> /BYTES/
+
+query T kvtrace
+INSERT INTO d VALUES(2, '[{"a": "b"}, 3, {"a": "b"}]')
+----
+CPut /Table/53/1/2/0 -> /TUPLE/
+InitPut /Table/53/2/Arr/3/2/0 -> /BYTES/
+InitPut /Table/53/2/Arr/"a"/"b"/2/0 -> /BYTES/
+
+query T kvtrace
+INSERT INTO d VALUES(3, '[{"a": [0,1,0]}, 3, {"a": "b"}]')
+----
+CPut /Table/53/1/3/0 -> /TUPLE/
+InitPut /Table/53/2/Arr/3/3/0 -> /BYTES/
+InitPut /Table/53/2/Arr/"a"/"b"/3/0 -> /BYTES/
+InitPut /Table/53/2/Arr/"a"/Arr/0/3/0 -> /BYTES/
+InitPut /Table/53/2/Arr/"a"/Arr/1/3/0 -> /BYTES/
+
+# Make sure that inserting NULL doesn't make an index entry.
+query T kvtrace
+INSERT INTO d VALUES(4, NULL)
+----
+CPut /Table/53/1/4/0 -> /TUPLE/
+
```

Comment:
I'd also add tests that you can delete and update with a null value and an inverted index

Reply:
done.